src/Compiler/Microsoft.Net.Compilers.Razor.Toolset/Microsoft.Net.Compilers.Razor.Toolset.csproj

```diff
@@ -0,0 +1,42 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
How do we validate that this package is functioning properly? Consider there is manual asset management below. That can get out of date over time. 

In Roslyn we effectively validate these packages by dogfooding them during build. Can we do a similar thing here? Do we have razor code here that could benefit from  a bootstrap?

Reply:
> Do we have razor code here that could benefit from a bootstrap?

I'm not aware of any. Open to suggestions on dogfooding; maybe we use the integration tests somehow?

Reply:
This is tricky in-repo because we'd need to reference the built package. The 'build real world code' repo would need to consume this though so we should get coverage from that. Also, we should definitely invest in a razor build boss that we can add checks like the two packages contain the same assemblies.